src/Commands/RemoveNonHelsinkiTPRUnitsCommands.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\helfi_platform_config\Commands;
+
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drush\Commands\DrushCommands;
+
+/**
+ * A Drush command file.
+ */
+final class RemoveNonHelsinkiTPRUnitsCommands extends DrushCommands {
+
+  /**
+   * The entity type manager.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected EntityTypeManagerInterface $entityTypeManager;
+
+  /**
+   * Constructs a new instance.
+   *
+   * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entity_type_manager
+   *   The entity type manager.
+   */
+  public function __construct(EntityTypeManagerInterface $entity_type_manager) {
+    $this->entityTypeManager = $entity_type_manager;
+  }
+
+  /**
+   * Removes all the TPR Unit entities which are not located in Helsinki.
+   *
+   * @command helfi:remove-non-helsinki-tpr-units
+   */
+  public function remove() : void {
+    // Get TPR Unit entities that DON'T have "Helsinki" or "Helsingfors" in
+    // their 'address__locality' field.
+    $entityStorage = $this->entityTypeManager->getStorage('tpr_unit');
+
+    $unit_ids = $entityStorage->getQuery()
+      ->condition('address__locality', 'Helsinki', '!=')
+      ->condition('address__locality', 'Helsingfors', '!=')
+      ->execute();
+
+    $units = $entityStorage->loadMultiple($unit_ids);
+
+    // Set up content lock service.
+    $lock_service = \Drupal::service('content_lock');
+
+    $unit_count = 0;
+
+    // Delete the units.
+    foreach ($units as $unit) {
+      \Drupal::messenger()->addMessage('Deleting "' . $unit->label() . '"');
+
+      // Release content lock if needed.
+      if ($lock_service->fetchLock($unit->id(), $unit->language()->getId(), NULL, 'tpr_unit')) {
+        $lock_service->release($unit->id(), $unit->language()->getId(), NULL, NULL, 'tpr_unit');
```

Comment:
Use named arguments here to skip optional/default paramers (https://www.php.net/manual/en/functions.arguments.php#functions.named-arguments):
`$lock_service->release($unit->id(), $unit->language()->getId(), entity_type: 'tpr_unit')`